building/tracks/syllabus/README.md

```diff
@@ -0,0 +1,165 @@
+The explanations are only there so that the student can do what needs to be done.
+
+We want to allow students to start writing code immediately, without having to understand everything up front.
+In order to achieve this we hand-wave over details and we leave a lot of things unexplained.
+We simplify and provide code stubs where possible.
+This reduces the cognitive burden of getting started and provides the time and space for the knowledge to sink in.
+By taking this approach we're not saying that the student doesn't need to know these things, we're saying that they don't need to know them _yet_.
+
+## Basic structure
+
+Exercises are structured as a tree with an introductory exercise at the top as the starting point.
+Later exercises teach concepts that depend on having understood concepts that are taught earlier.
+
+## Porting and borrowing
+
+It can be worth looking at how other language tracks have built out their concept exercises.
+You can find examples of Concept Exercises from other language tracks [here](https://exercism.org/docs/building/tracks/stories).
+
+That said, if you decide to use other exercises as a starting point for your own, be careful to ensure that the resulting exercise is about the concept as it exists in your language.
+Sometimes concepts differ subtly, sometimes radically.
+Sometimes concepts don't exist at all.
+
+The syllabus, and therefore the concept tree, should represent the concepts that exist in the language.
+
+Don't include concepts just because other tracks do.
+
+In some cases it might be tempting to put a concept in because people often have to work around it by using concepts that do exist.
+Rather than doing this, introduce the concept that the language _does_ use, and consider adding an exercise that explains how to use it in that type of situation.
+
+For example, in Go there are no enums.
+Instead the Go concept tree introduces constants, and teaches how to use constants in the type of situation where you might use enums in other languages.
+
+## Asking for help
+Don't hesitate to ask for help.
+It's better to ask up front or while working on an exercise rather than discussing during code review.
+
+## Getting started
+
+Our experience has taught us that the most pragmatic way to develop a syllabus is to grow the concept tree organically, starting with the simplest concepts.
+We don't have to get everything right up front, but it's really useful to not think too far ahead.
+
+We start with concepts that are easy.
+
+Some things are easy because they are inherently simple.
+Other things are not all that simple, but feel easy because they are familiar.
+Familiar is good.
+Familiar is not confusing.
+
+While the endpoint is to write idiomatic code, the stepping stones to get there are not always idiomatic.
+Using what is familiar—even if it is not a great example of code in that language—helps move a student more quickly toward the goal of code that is more typical of the language.
```

Comment:
It's not supposed to have spaces. When it's rendered in proper text as opposed to a single-spaced font the em-dash is much wider, which helps readability. Would you do a quick check in the "view file" view to see if that works? If not I'd rather rework to not use dashes than to have spaces around them.

Reply:
I don't see the spacing on my screen: https://github.com/exercism/docs/blob/e3fbb33605b5296260c73a8072ad7495b759031e/building/tracks/syllabus/README.md

Reply:
Yeah, no spacing, but a much wider dash.

Reply:
I would rather we have spacing as I find it really hard to read without them. As I understand it styles for long em varies and there's no absolute rule (source [merriam webster](https://www.merriam-webster.com/words-at-play/em-dash-en-dash-how-to-use#:~:text=Spacing%20around%20an%20em%20dash,right%20up%20next%20to%20it) so I'd rather the extra clarity of spaces.

Reply:
I had no idea! I think maybe I've read a newspaper less than ten times in my life, and can read hundreds of books in a year, so that would explain my assumptions :)

Spaces it is!